db/migrate/20160719131003_create_recipes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRecipes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :recipes do |t|
+      t.string :recipe_name
+      t.string :ingredient_list, array: true, default: []
+      t.text :description
+      t.string :img_link
```

Comment:
more, image link, but having images can be another ticket, will be a lot easier, because it's complex topic. need more just single file if you want upload them, not only put link